src/ui/VirtualFileRenderer/ColorBar.tsx

```diff
@@ -0,0 +1,44 @@
+interface CoverageBarProps {
+  locationHash?: string
+  lineNumber: number
+  coverage?: 'H' | 'M' | 'P'
+}
+
+// exporting for testing purposes
+export const ColorBar = ({
+  coverage,
+  locationHash,
+  lineNumber,
+}: CoverageBarProps) => {
+  if (locationHash && locationHash === `#L${lineNumber}`) {
```

Comment:
You can just compare to the string literal
```suggestion
  if (locationHash === `#L${lineNumber}`) {
```